spec/features/idv/in_person_spec.rb

```diff
@@ -27,7 +27,7 @@
       complete_prepare_step(user)
 
       # location page
-      complete_location_step
+      complete_full_address_location_step
```

Comment:
now, the "full address" location step is our only location step, right? Should we rename the helper methods and keep these specs otherwise unchanged?